.vscode/launch.json

```diff
@@ -18,6 +18,20 @@
         "0"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Run repository mocha tests",
+      "program": "${workspaceRoot}/packages/build/node_modules/.bin/_mocha",
+      "cwd": "${workspaceRoot}",
+      "args": [
+        "--opts",
+        "${workspaceRoot}/test/mocha.opts",
+        "packages/repository/dist8/test/**/*.js",
```

Comment:
Hmm, this will not work for people who are running Node 10.x (e.g. myself). Proposed solution: include Node version in the task name, e.g. `Run repository tests from dist8`. What would be even better: assuming people build only the single target matching their current Node version, we can replace `dist8` with `dist*`.

```
"packages/repository/dist*/test/**/*.js",
```

Reply:
We pushed these changes in `launch.json` to share it while we pair program, so it should be safe to ignore. We will remove before merging when the code is finalized.